resources/license.rb

```diff
@@ -1,10 +1,12 @@
 property :license, String, name_property: true, required: true
+property :key, String
```

Comment:
We don't need a `:key` property. We're already processing that in the `:license` property above. Take a look at the readme for how we're handling that.